Gemfile

```diff
@@ -28,6 +28,10 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Discussion: https://github.com/18F/college-choice/issues/597#issuecomment-139034834
 gem "puma", "~> 2.16.0"
 
+#redis and redis-namespace for session and mvi persistence
+gem 'redis'
```

Comment:
can we add a version that maps to the redis version that is supported by GovCloud
from docs : Redis Version 2.8.24
http://docs.aws.amazon.com/AmazonElastiCache/latest/UserGuide/SelectEngine.html